src/django/api/serializers/facility/utils.py

```diff
@@ -410,6 +410,9 @@ def regroup_claims_for_sector_field(claims: list,
 
 
 def format_date(date: str) -> str:
+    if not date:
```

Comment:
I doubt that None passed here is a normal situation. I think we need to dig deeper why it happened here. I expect a problem with logic in some other place